packages/swapper/src/swappers/zrx/getZrxTradeQuote/getZrxTradeQuote.ts

```diff
@@ -23,32 +23,25 @@ export async function getZrxTradeQuote(
   if (buyAsset.chain !== ChainTypes.Ethereum || sellAsset.chain !== ChainTypes.Ethereum) {
     throw new ZrxError('getQuote - Both assets need to be on the Ethereum chain to use Zrx')
   }
-  if (!sellAmount && !buyAmount) {
-    throw new ZrxError('getQuote - sellAmount or buyAmount amount is required')
+  if (!sellAmount) {
+    throw new ZrxError('getQuote - sellAmount is required')
```

Comment:
As far as I can tell, `sellAmount` is never falsy, I'm not sure what this is for.

Reply:
I agree. will remove